app/Policies/CommentPolicy.php

```diff
@@ -57,11 +57,8 @@ public function create(User $user, ?Model $commentable = null): bool
             return false;
         }
 
-        // users cannnot comment on the wall of a user who has them blocked
         if ($commentable !== null && $commentable instanceof User) {
-            if ($commentable->isBlocking($user)) {
-                return false;
-            }
+            return (new UserCommentPolicy())->create($user, $commentable);
```

Comment:
Shouldn't this be `$user->can('create', [UserComment::class, $commentable])` or something similar?

Reply:
It should. It didn't even occur to me that we had a `UserComment` class. Updated in latest.